### ISSUE TITLE
Parametrized (in terms of type) ParametrizedAffExpr

### ISSUE THREAD

**Issue by jd-lara** [JuliaStochOpt/ParameterJuMP.jl#39]
Currently we use this container for `JuMP.AffineExpression` in order to hold expressions of any type of VariableRef

```Julia
const JuMPAffineExpressionArray = Matrix{JuMP.GenericAffExpr{Float64,V}} where V <: JuMP.AbstractVariableRef
```
However, `ParameterJuMP.ParametrizedAffExpr` only seems to take the type of the number and not the type of variables which leads to the defintion

```Julia
const JuMPParamAffineExprArray = Matrix{ParameterJuMP.ParametrizedAffExpr{Float64}}
```

Is there a way to enable `AbstractVariableRef` parametrization in `ParametrizedAffExpr{Float64,V}`?

**Comment by joaquimg** [comment]
yes, we can change this by redefining:
```julia
mutable struct ParametrizedAffExpr{C, V} <: JuMP.AbstractJuMPScalar
    v::JuMP.GenericAffExpr{C,V}
    p::JuMP.GenericAffExpr{C,Parameter}
end
```
Some other parts of the code will need minor modifications

**Comment by joaquimg** [comment]
solved by #45
